test/server/graphql/v1/collective.test.js

```diff
@@ -735,6 +767,34 @@ describe('graphql.collective.test.js', () => {
       expect(members[0].member.slug).to.equal('piamancini');
       expect(members).to.have.length(1);
     });
+
+    it('gets totalAmountSpent by collective', async () => {
+      const query = `query TotalCollectiveContributions($slug: String, $type: String) {
+        Collective(slug: $slug) {
+          id
+          currency
+          stats {
+            id
+            totalAmountSpent
+          }
+          transactions(type: $type) {
+            currency
+            netAmountInCollectiveCurrency
+          }
+        }
+      }`;
+
+      const result = await utils.graphqlQuery(query, {
+        slug: 'olu',
+        type: 'DEBIT',
+      });
+
+      result.errors && console.error(result.errors);
+      expect(result.errors).to.not.exist;
+      const collective = result.data.Collective;
+      const totalAmountSpent = 1000 * 1.1 + 1000;
```

Comment:
@flickz didn't you say [here](https://github.com/opencollective/opencollective-api/pull/3039/files#r357670268) that hard-coding wasn't the right approach because a slight change in the value of exchange rate will cause the test to break?

Reply:
True but I realized the rate was hardcoded for test in https://github.com/opencollective/opencollective-api/blob/master/server/lib/currency.js#L76